test/built-ins/Temporal/Calendar/prototype/weekOfYear/argument-propertybag-calendar-string.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.weekofyear
+description: A calendar ID is valid input for Calendar
+features: [Temporal]
+---*/
+
+const instance = new Temporal.Calendar("iso8601");
+
+const calendar = "iso8601";
+
+const arg = { year: 1976, monthCode: "M11", day: 18, calendar };
+const result = instance.weekOfYear(arg);
+assert.sameValue(result, 47, `Calendar created from string "${arg}"`);
```

Comment:
Perhaps we should some something more like 

````
`Calendar created from ${typeof arg} "${arg}"`
````

? In this case, I guess it's not a string.

Reply:
I think I intended it to be `Calendar created from string ${calendar}` in all these tests.

Reply:
(at least, the property bag ones)